back-end/src/sciler/config/checkConfig.go

```diff
@@ -192,12 +193,32 @@ func checkActionInstructionType(valueType reflect.Kind, instructionType string,
 		if valueType != reflect.Slice {
 			return fmt.Errorf("level III - implementation error: on rule %s's actions: instruction type array/slice expected but %s found as type of the value: %v", ruleID, valueType.String(), value)
 		}
+	case "status":
+		if valueType != reflect.String {
+			return fmt.Errorf("level III - implementation error: on rule %s's actions: instruction type status expected but %s found as type of the value: %v", ruleID, valueType.String(), value)
+		}
+		return config.checkStatusInstructionType(value.(string), ruleID)
 	default:
 		return fmt.Errorf("level III - implementation error: on rule %s's actions: custom types of value like: %s, are not yet implemented", ruleID, instructionType)
 	}
 	return nil
 }
 
+// checkStatusInstructionType checks if a status is in the form: `deviceID.component`
+// and if the device and component exists
+func (config *WorkingConfig) checkStatusInstructionType(status string, ruleID string) error {
+	split := strings.Split(status, ".")
+	if len(split) == 2 {
+		deviceID := split[0]
+		componentID := split[1]
+		device := config.Devices[deviceID]
+		if device != nil && (device.Input[componentID] != "" || device.Output[componentID].Type != "") {
```

Comment:
can you split these checks and return a specific error message

Reply:
done